src/mbgl/tile/raster_dem_tile.hpp

```diff
@@ -0,0 +1,104 @@
+#pragma once
+
+#include <mbgl/tile/tile.hpp>
+#include <mbgl/tile/tile_loader.hpp>
+#include <mbgl/tile/raster_dem_tile_worker.hpp>
+#include <mbgl/actor/actor.hpp>
+
+namespace mbgl {
+
+class Tileset;
+class TileParameters;
+class HillshadeBucket;
+
+enum class DEMTileNeighbors : unsigned char {
```

Comment:
Nit: Can we use `uint8_t` to be in line with how we use this type in other parts of the codebase?